django/chat/templates/chat/modals/presets/presets_form.html

```diff
@@ -1,34 +1,40 @@
 {% load i18n %}
 
+{% csrf_token %}
```

Comment:
I think this is in 2 places. You can remove it from here, or the other place (line 38).